src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMetaTileEntity_SolarTower.java

```diff
@@ -80,10 +81,14 @@ protected final GT_Multiblock_Tooltip_Builder createTooltip() {
 		.addInfo("Contributing Green Energy towards the future")
 		.addInfo("Input: "+MISC_MATERIALS.SOLAR_SALT_COLD.getLocalizedName())
 		.addInfo("Output: "+MISC_MATERIALS.SOLAR_SALT_HOT.getLocalizedName())
-		.addInfo("Surround with rings of Solar Heaters")
-		.addInfo("Each ring increases tier")
+		.addInfo("Surround with rings of Solar Reflectors")
+		.addInfo("The Reflectors increase the internal heat value of the Tower")
+		.addInfo("Converting Cold to Hot Solar Salt reduces heat, equal to the amount converted")
+		.addInfo("Each Reflector ring increases tier")
 		.addInfo("1 = 1, 2 = 2, 3 = 4, 4 = 8, 5 = 16")
-		.addInfo("Tier decreases heating time and allows more salt to be processed")
+		.addInfo("Each tier gives an additional bonus to all Reflectors")
+		.addInfo("The heat increase is more efficient at exactly half of maximum heat")
+		.addInfo("Minimum efficiency at 0 or 100000 heat, maximum efficiency at 50000")
 		.addSeparator()
 		.beginStructureBlock(6, 27, 6, false)
```

Comment:
Height is actually 28. Width and length are 9, not including reflectors

Reply:
Set minimum and maximum width and length to 1 ring and 5 rings.